cmd/peek-submission/main_test.go

```diff
@@ -22,30 +22,38 @@ func TestPeekBase(test *testing.T) {
 		targetSubmission    string
 		expectedSubmimssion string
 		expectedExitCode    int
+		expectedLocator     string
+		logLevel            string
 	}{
-		{"course-student@test.edulinq.org", "course101", "hw0", "", "1697406272", 0},
-		{"course-student@test.edulinq.org", "course101", "hw0", "1697406272", "1697406272", 0},
-		{"course-student@test.edulinq.org", "course101", "hw0", "course101::hw0::student@test.com::1697406256", "1697406256", 0},
+		{"course-student@test.edulinq.org", "course101", "hw0", "", "1697406272", 0, "", ""},
+		{"course-student@test.edulinq.org", "course101", "hw0", "1697406272", "1697406272", 0, "", ""},
+		{"course-student@test.edulinq.org", "course101", "hw0", "course101::hw0::student@test.com::1697406256", "1697406256", 0, "", ""},
 
-		{"course-admin@test.edulinq.org", "course101", "hw0", "", "", 0},
-		{"course-student@test.edulinq.org", "course101", "hw0", "ZZZ", "", 0},
+		{"course-admin@test.edulinq.org", "course101", "hw0", "", "", 0, "", ""},
+		{"course-student@test.edulinq.org", "course101", "hw0", "ZZZ", "", 0, "", ""},
 
-		{"course-student@test.edulinq.org", "ZZZ", "hw0", "", "", 2},
-		{"course-student@test.edulinq.org", "course101", "ZZZ", "", "", 2},
-		{"course-student@test.edulinq.org", "ZZZ", "ZZZ", "", "", 2},
+		{"course-student@test.edulinq.org", "ZZZ", "hw0", "", "", 2, "-018", "error"},
+		{"course-student@test.edulinq.org", "course101", "ZZZ", "", "", 2, "-022", "error"},
+		{"course-student@test.edulinq.org", "ZZZ", "ZZZ", "", "", 2, "-018", "error"},
 	}
 
+	util.ShouldExit = false
+	defer func() {
+		util.ShouldExit = true
+	}()
+
+	oldExitCode := util.GetLastExitCode()
+	defer func() {
+		util.SetExitCode(oldExitCode)
+	}()
+
 	for i, testCase := range testCases {
 		args := []string{
 			testCase.targetEmail,
 			testCase.courseID,
 			testCase.assignmentID,
 			testCase.targetSubmission,
-		}
-
-		var exitCode int
-		util.Exit = func(code int) {
-			exitCode = code
+			"--log-level", testCase.logLevel,
```

Comment:
Why does the log level need to be set?
Can't we just set the level to fatal for this test (and thereby ignore all the logs)?

Or maybe even do it in internal/cmd, like we do in internal/db:
https://github.com/edulinq/autograder-server/blob/main/internal/db/test.go#L31